vault/data_source_aws_access_credentials.go

```diff
@@ -69,6 +69,12 @@ func awsAccessCredentialsDataSource() *schema.Resource {
 				Optional:    true,
 				Description: "ARN to use if multiple are available in the role. Required if the role has multiple ARNs.",
 			},
+			"region": {
```

Comment:
Will this introduce regressions since `creds` won't be set on this data source?  I'm unsure where region was being pulled from prior to this patch.

Reply:
Well, I don't know. It has a default set, which I assume means it would cause errors for anyone that wasn't in that region, but I also am of the opinion that it wasn't getting data about what region it should be in before this, and somehow was able to just share data between non-private datacenters (i.e. non us-gov or cn datacenters) but I'm not sure.